python/ray/autoscaler/aws/example.yaml

```diff
@@ -22,7 +22,7 @@ idle_timeout_minutes: 5
 provider:
     type: aws
     region: us-west-2
-    # availability_zone: us-west-2a
+    availability_zone: default
```

Comment:
Will this work if `availability_zone` is not provided? I don't think it should be required.

If it is required, then you'll also need to change `development_example.yaml`.

Reply:
PS: some availability zones may not contain any instances of desired type. Perhaps t2/c5 may be in all instances, but I've seen error this with other instance types

Reply:
You get error like this `botocore.exceptions.ClientError: An error occurred (Unsupported) when calling the RunInstances operation: Your requested instance type (g3.4xlarge) is not supported in your requested Availability Zone (us-east-1f). Please retry your request by not specifying an Availability Zone or choosing us-east-1b, us-east-1a, us-east-1c, us-east-1e.`